app/models/run.rb

```diff
@@ -159,6 +159,22 @@ def publish_aging
     publish_age_every(:day, 30)
   end
 
+  def playtime_between_pbs
```

Comment:
Is this method called anywhere?

Reply:
Oops nope, leftover from before I decided to do this in JS.